docs/lsm/attributes_metadata/attributes_metadata.md

```diff
@@ -21,13 +21,13 @@ entity Interface:
 
         :attr interface_name: The name of the interface
     """
-    string interface_name 
+    string interface_name
 end
 ```
 
 A detailed example can be found {ref}`here <quickstart_orchestration_model>`.
 
-
+(attributes_metadata_attribute_modifiers)=
```

Comment:
Fixes the broken link in `More information on attribute modifiers can be found here.` in this paragraph: https://docs.inmanta.com/inmanta-service-orchestrator/8/lsm/quickstart/quickstart.html#orchestration-model